src/main/java/me/totalfreedom/totalfreedommod/command/Command_say.java

```diff
@@ -9,7 +9,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.ADMIN, source = SourceType.BOTH)
-@CommandParameters(description = "Broadcasts the given message as the server, includes sender name.", usage = "/<command> <message>")
+@CommandParameters(description = "Broadcasts the given message to get peoples attention, includes sender name.", usage = "/<command> <message>")
```

Comment:
Not sure this is an accurate change? You are making a server announcement...

Reply:
I feel like that's more what you use /announce for, but I can see that as well.

Reply:
fixed